functions/storage/jira_id_storage.json

```diff
@@ -0,0 +1 @@
+{"stevenbrand": "5fac6667248fc30078dafbee", "dacorredor": "5fac69d771ab9e006a43937c", "melandres": "5fac65129e35e50068d08759"}
```

Comment:
We should use another option for this info so as not to expose it. Maybe a DB or stuff like that.